src/style.css

```diff
@@ -0,0 +1,24 @@
+
+body {
+  margin: 0;
+  padding: 0;
+}
+
+.stars {
+  display: flex;
+}
+
+.star__stella {
```

Comment:
what is stella?)
```suggestion
.star__star {
```

Reply:
![image](https://user-images.githubusercontent.com/36257947/106751949-92aad800-6632-11eb-8d22-dd72a0df779a.png)